examples/charts/unauthenticated/README.md

```diff
@@ -57,17 +57,19 @@ This sample is preconfigured to render a specific chart. You can run the sample
 
 5. Make sure the "External users can view data in this data source" option is toggled on and "Allow unauthenticated data access" has been selected.
 
+6. Go to the Data Sources tab, find the data source that you are using on the chart, and choose External Sharing Options from the ... menu. Make sure that embedding is enabled for this data source and select '**Unauthenticated or Verified Signature**'
+
 ### Enable chart for embedded access
 
-6. Find the chart you want to embed, click the **...** menu and select **Embed chart**
+7. Find the chart you want to embed, click the **...** menu and select **Embed chart**
 
-7. Ensure the **Unauthenticated** tab is selected and turn on '**Enable unauthenticated access**'
+8. Ensure the **Unauthenticated** tab is selected and turn on '**Enable unauthenticated access**'
 
-8. Select the **Javascript SDK** option
+9. Select the **Javascript SDK** option
 
-9.  Note the **Chart ID** and the **Base URL**, as you will need them for running the demo.
+10. Note the **Chart ID** and the **Base URL**, as you will need them for running the demo.
 
-10. **Optional**
+11. **Optional**
    In the same menu, note the **User Specified Filters** option. If you wish to try out filtering on your own dataset, you will need to whitelist a field by which to filter on. For example, our sample AirBnB dataset filters on `address.country`.
```

Comment:
Nit: While we are here, can we change the "whitelist" to "allowed list" to reflect the current product state? This can be done separately too.